bridge_adaptivity/api/backends/api_client.py

```diff
@@ -109,12 +109,12 @@ def get_available_courses(request, source_ids=None):
                 )
             )
         except HttpClientError as exc:
-            errors.append(
-                _(
-                    "Content source '{}' is unavailable. Reason: {} Please contact Bridge administrator to solve the"
-                    " problem."
-                ).format(content_source.name, str(exc))
-            )
+            error_list = list()
+            if str(exc) in errors:
```

Comment:
Creating a new variable with value `str(exc)` suits better for cases when we use it more than one time.